### ISSUE TITLE
package unavailable

### ISSUE THREAD

**Issue by duncdrum** [mandoku/mandoku#26]
Trying to install on os x 10.11.6 on 
`GNU Emacs 25.1.1 (x86_64-apple-darwin15.6.0, Carbon Version 157 AppKit 1404.47)
 of 2016-08-28`
I receive the following error:

``````
installing org-plus-contrib
package-compute-transaction: Package ‘org-plus-contrib-’ is unavailable```
``````

**Comment by cwittern** [comment]
I just tried to reproduce the issue here.  
TL;DR: The issue is caused by an older version of the mandoku package, which depends on org-plus-contrib, but that package was later removed from melpa.  We now use the standard org, which is also mentioned as dependence in the current mandoku package. 

Here is what works: I used the most recent Emacs app from github on a fresh 10.11, which is the RC2 for 25.1 and pasted the following into the scratch buffer:
(progn
(require 'package)
(package-initialize)
(add-to-list 'package-archives
             '("melpa" . "http://melpa.org/packages/"))
(package-refresh-contents)
(package-install 'mandoku)
)
Evaluating this with C-x e will start the installation procedure.  Once it is done, you can verify that the installation was successful by calling up the catalog with "M-x mandoku-show-catalog"  If you run this the first time, there are some further steps to follow to establish a local place for the files on your computer and downloading the Workspace from github. 

Please try this and tell me if you run into further problems.

**Comment by duncdrum** [comment]
@cwittern  the new install routine has worked but on starting emacs I now have an error message concerning the ```mandoku init file``` being missing. Looks like something I ll be able to fix after getting into the emacs vibe. Thanks for the help.

**Comment by cwittern** [comment]
Well, this is probably due to traces of an older version of mandoku still lying around.  If you figure out what exactly went wrong, please post it here so that others who might run into the same problem can find it .  
My guess is that a file mandoku-init.el is still referenced from your emacs init file but not available (and not needed) any more.  I know all this is pretty messy, but installation is hard, and updating it (sometimes in various stages of failure) is still harder... A last resort is always a clean start.

**Comment by duncdrum** [comment]
fixed in the iterim
